lib/gooddata/lcm/actions/synchronize_etls_in_segment.rb

```diff
@@ -81,13 +81,42 @@ def call(params)
             end
           end
 
-          params.synchronize.peach do |info|
+          delete_extra_process_schedule = GoodData::Helpers.to_boolean(params.delete_extra_process_schedule)
+
+          params.synchronize.peach do |info| # rubocop:disable Metrics/BlockLength
+            if delete_extra_process_schedule
+              from_project = client.projects(info.from) || fail("Invalid 'from' project specified - '#{info.from}'")
+              from_project_processes = from_project.processes
+              from_project_process_id_names = Hash[from_project_processes.map { |process| [process.process_id, process.name] }]
+              from_project_process_names = from_project_processes.map(&:name)
+              from_project_schedule_names = from_project.schedules.map { |schedule| [schedule.name, from_project_process_id_names[schedule.process_id]] }
+            end
+
             to_projects = info.to
             to_projects.peach do |entry|
               pid = entry[:pid]
               to_project = client.projects(pid) || fail("Invalid 'to' project specified - '#{pid}'")
+
+              if delete_extra_process_schedule
```

Comment:
Would it be better to name the parameter `delete_extra_processes_and_schedules` or `delete_extra_etl`?